package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "kafka-avro",
-  "version": "1.0.2",
-  "main": "./lib/kafka-avro",
+  "version": "1.0.3",
```

Comment:
you dont need to change the dump - it will change when I release to master. Just reset this change and we can release it